actions/cabal-ghc-compat/action.yml

```diff
@@ -4,14 +4,14 @@ description: >
   Cabal library bundled with GHC.
 inputs:
   dirs:
-    description: Directories in which to run
+    description: Directories in which to run (relative to the repo root)
     required: false
     default: "."
   ghc:
     description: GHC version, e.g., "9.10.1"
     required: true
   pkgs:
-    description: Extra packages to make available in the Nix shell
+    description: Extra packages to make available in the Nix shell (separated by spaces)
```

Comment:
I'd still recommend some examples or clarification here: should I be specifying the package by nix attribute name (e.g. `zlib`), by actual attrset (e.g. `pkgs.zlib`), by pname (e.g. "zlib"), by name (e.g. "zlib-3.2"), or some other method?  (The first, I know, but we want to tell the user that).

Reply:
Good call, added an example from What4 and referred from the README.